### PR TITLE
Fix vulkan api data race and crash when destroyed

### PR DESCRIPTION
**Pull request by SaumonDesMers**

### CODE REVIEW

File: src/app/threads/AThreadWrapper.hpp
Comment (afaure42) [inline src/app/threads/AThreadWrapper.hpp:1]: Why is jthread no longer protected ?

**Review by afaure42 — CHANGES_REQUESTED** [review]
Jthread is no longer protected in AThreadWrapper, is it intended ?

File: src/app/threads/AThreadWrapper.hpp
Comment (SaumonDesMers) [reply src/app/threads/AThreadWrapper.hpp:1]: That was for testing porpose, I'll get it back to protected

**Review by afaure42 — APPROVED** [review]
👍

**Pull request merged**